<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gps</class>
 <widget class="QDialog" name="gps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>1296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gps</string>
  </property>
  <widget class="QPushButton" name="ButtonStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStop">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableSvInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>661</width>
     <height>1001</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1120</y>
     <width>661</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>No Fix</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
